apps/collabora_docker.sh

```diff
@@ -23,7 +23,19 @@ lowest_compatible_nc 13
 ram_check 2 Collabora
 cpu_check 2 Collabora
 
-# Check if collabora is already installed
+# Check if Nextcloud is installed with TLS
+if ! occ_command_no_check config:system:get overwrite.cli.url | grep -q "https"
+then
+msg_box "Sorry, but Nextcloud needs to be run on HTTPS which doesn't seem to be the case here.
+You easily activate TLS (HTTPS) by running the Let's Encrypt script.
+More info here: https://bit.ly/37wRCin
+
+To run this script again, just exectue 'sudo bash $SCRIPTS/menu.sh' and choose:
+Additional Apps --> Collabora."
+    exit
+fi
+
+# Check if Collabora is already installed
```

Comment:
Since we use this in multiple places, please make it a function.

Maybe something like `check_nextcloud_https`?

Reply:
done